androbot/actions.py

```diff
@@ -171,3 +171,9 @@ def add_question_score(self, question_id: int, tg_user_id: int, score: int) -> Q
 
     def get_question_score(self, question_id: int, tg_user_id: int) -> List[QuestionScore]:
         return crud.get_question_score(self.db, question_id, tg_user_id)
+
+    def get_questions_scores(self, tg_user_id: int) -> List[QuestionScore]:
```

Comment:
```suggestion
    def get_all_questions_scores(self, tg_user_id: int) -> List[QuestionScore]:
```
Может так назовем? А то я потупил сначала